src/main/java/io/cdap/plugin/gcp/bigquery/sink/AbstractBigQuerySink.java

```diff
@@ -150,7 +152,12 @@ protected final void initOutput(BatchSinkContext context, BigQuery bigQuery, Str
     List<String> fieldNames = fields.stream()
       .map(BigQueryTableFieldSchema::getName)
       .collect(Collectors.toList());
-    BigQuerySinkUtils.recordLineage(context, outputName, tableSchema, fieldNames);
+    String fqn = BigQueryUtil.getFQN(getConfig().getDatasetProject(),
+                                               getConfig().getDataset(), tableName);
+    Asset.Builder assetBuilder = Asset.builder(outputName);
```

Comment:
nit: let's build all fields of the asset in one line instead of splitting it between here and when we pass it as an arg to recordLineage()?
Here and in other places